packages/app/src/hooks/useAutoCompleteOptions.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
upgrade guide recommends adding `use client` for all hooks that use react's hooks so that if it were ever used in server side rendering, it wouldn't error out.